server/simulation/random.js

```diff
@@ -32,6 +32,14 @@ const randomRating = () => {
   return parseFloat(rating);
 };
 
+/**
+ *   
```

Comment:
Remove empty line.

Yes, it is a bit silly. But these comments are later parsed to create API documentation automatically and are also displayed by the IDE, so it is important to keep to a standard.